lib/components/Select.js

```diff
@@ -17,83 +18,86 @@ const filterProps = (props) => {
 /**
  * The Select component. Renders to a plain HTML <select>.
  */
-export default React.createClass({
-  displayName: 'Select',
-
-  propTypes: {
-    onChange: React.PropTypes.func.isRequired,
-    options: React.PropTypes.array.isRequired,
-    placeholder: React.PropTypes.string,
-    value: React.PropTypes.any,
-  },
-
-  handleChange(e) {
-    let value;
-
-    if (this.isMultiSelect()) {
-      value = toArray(e.target.options)
-        .filter(option => option.selected)
-        .map(option => option.value);
-    } else {
-      value = e.target.value;
+
+const propTypes = {
+    onChange: PropTypes.func.isRequired,
+    options: PropTypes.array.isRequired,
+    placeholder: PropTypes.string,
+    value: PropTypes.any,
+};
+
+class Select extends React.Component {
+    handleChange(e) {
```

Comment:
I'm surprised there is no need to `bind` this function and the others in the class in the constructor, but have yet to try running the code.